### PR TITLE
Add `build` command to build a specific version of a package on warehouse 

### PR DESCRIPTION
**Pull request by wjarvis1**
Add `build` command to build a specific version of a package on warehouse

## Summary

Need a `build` command in the cli to build a specific version of a package on warehouse.
## Changelog

Add `build` command to build a specific version of a package on warehouse, and optionally promote with -m flag

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,7 +117,7 @@ DESCRIPTION
   If no version is specified, the head version will be returned.
 ```
 
-_See code: [src/commands/get/build.js](https://github.com/warehouseai/wrhs/blob/v0.3.1/src/commands/get/build.js)_
```
Comment (wjarvis1) [inline README.md]: the v0.3.1 links were all broken

Comment (fritzmonkey) [reply README.md]: These are generated by oclif. Looks like we tag as `0.3.1` and they are expecting `v0.3.1`. It looks like there is some support for configuring this, but I would have to dig more. We might want to just tag this repo with the `v`.
https://github.com/oclif/dev-cli/blob/v1.22.1/src/commands/readme.ts#L171

File: src/base.js [hunk 2]
```diff
@@ -70,6 +70,15 @@ class WrhsCommand extends Command {
     return `Missing warehouse ${isStatus ? 'status ' : ''}host. Please configure a \`~/.wrhs\` config file or use the \`--${isStatus ? 'status-' : ''}host\` option.\n ${seeTheDocsMessage}`;
   }
 
+  /**
+   * Builds a missing version error message
+   *
+   * @returns {string} The missing host error message
```
Comment (fritzmonkey) [inline src/base.js]:
```suggestion
   * @returns {string} The missing version number error message
```

File: src/commands/build.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+        if (err) {
+          this.renderError('build', args.package, err);
+          return reject(err);
+        }
+        if (flags.json) {
+          this.log(JSON.stringify(response));
+          return resolve(JSON.stringify(response));
+        }
+
+        this.log('Build triggered successfully. See `get:status` command to get the status of a build.');
```
Comment (fritzmonkey) [inline src/commands/build.js]: Since we have the information, we should build the full `wrhs get:status` command for easy copying.

**Pull request merged**